cmd/testdata/gql_orb_list/pretty_json_output.json

```diff
@@ -2,6 +2,7 @@
   "orbs": [
     {
       "name": "circleci/codecov-clojure",
+      "version": "0.0.3",
```

Comment:
This shows up because we're tagging the `HighestVersion` field in json as `"version"`, so that deserializing the JSON response from the API which is normally `orb { version }`.

Since we set this field when listing orbs, it also shows up when serializing the data back to JSON:
https://github.com/CircleCI-Public/circleci-cli/blob/b73cf303490208d69b0496954ceaea6dfbf9c659/api/api.go#L863